scripts/bumpfee.py

```diff
@@ -148,36 +145,54 @@ def sign_psbt(new_tx, old_tx, wallet_service):
 
     return btc.PartiallySignedTransaction.deserialize(signed_psbt)
 
-def create_bumped_tx(tx, fee_per_kb, wallet):
-    check_valid_candidate(tx, wallet)
+def create_bumped_tx(tx, fee_per_kb, wallet, output_index=-1):
+    check_valid_candidate(tx, wallet, output_index)
```

Comment:
Have I missed something, or wouldn't the idea be to wrap this call with a `try`, `except ValueError as e` and then use the error message and print it out (or better send to a user feedback function, but `jmprint` is fine for now), and then `sys.exit(EXIT_ARGERROR)`?

Reply:
Currently the only call to `check_valid_candidate` is here, and this function is wrapped in an appropriate try/check, so this is fine now.